app/src/main/java/org/glucosio/android/presenter/A1CCalculatorPresenter.java

```diff
@@ -72,7 +72,14 @@ public void checkGlucoseUnit() {
     }
 
     public void saveA1C(double a1c) {
-        HB1ACReading a1cReading = new HB1ACReading(a1c, new Date());
+        User user = dbHandler.getUser(1);
+        double finalA1c = a1c;
+        if (!"percentage".equals(user.getPreferred_unit_a1c())) {
+            GlucosioConverter converter = new GlucosioConverter();
+            finalA1c = converter.a1cIfccToNgsp(a1c);
```

Comment:
Do we have a dictionary somewhere? I understand a1c but what are CC and NGPS I don't know.